### ISSUE TITLE
Proposition : le libre chez MétéoFrance

### ISSUE THREAD

**Issue by pierreozoux** [blue-hats/ateliers#23]
17 Février
intervenants:
 - Remy Giraud (Météo France)
 - IndieHosters

(En préparation du texte de présentation)

**Comment by bzg** [comment]
Bonjour Pierre, très bonne idée !

N'hésitez pas à faire une PR directement quand vous avez la description et un intervenant de Météo France.

**Comment by pierreozoux** [comment]
L'intervenant est déjà ok. (Remy Giraud de Météo France)

La PR je la fais contre ce fichier?
https://github.com/blue-hats/ateliers/blob/main/ateliers.org

Si on peut déjà bloquer le slot, ça me va :)

**Comment by bzg** [comment]
Oui, vous pouvez bloquer la date, et oui la PR se fait contre ce fichier :)  (Attention, c'est du `.org`.)

**Comment by pierreozoux** [comment]
@bzg j'ai vu que c'était trop tard pour cette date, pas de soucis.

Mon contact chez MF n'est pas dispo le vendredi, est-ce qu'on peut faire une entorse à la regle?

Une bonne journée!

**Comment by pierreozoux** [comment]
Ces évènements existent toujours, je ne vois pas le lien vers le calendrier.

Si oui, je vais essayer de trouver une personne MF pour venir le faire avec moi.

Merci et bonne journée!
